Makefile.pre.in

```diff
@@ -1457,7 +1480,12 @@ libinstall:	build_all $(srcdir)/Modules/xxmodule.c
 		else	true; \
 		fi; \
 	done
-	@for d in $(LIBSUBDIRS); \
+	@if test "$(TEST_MODULES)" = yes; then \
+		subdirs="$(LIBSUBDIRS) $(TESTSUBDIRS)"; \
+	else \
+		subdirs="$(LIBSUBDIRS)"; \
+	fi; \
```

Comment:
The buildroot patch uses a different approach:

```
+ifeq (@TEST_MODULES@,yes)
+LIBSUBDIRS += $(TESTSUBDIRS)
+endif
```

IMO it's more generic. It avoids to have to declare subdirs twice. Would you mind to attemp this approach?

See: https://git.buildroot.org/buildroot/tree/package/python3/0013-Add-an-option-to-disable-installation-of-test-module.patch

Reply:
When I go through the messages of https://bugs.python.org/issue20210, I found this syntax(ifeq) can't be recognized by FreeBSD make. I think this explains why I can't find the occurrence of ifeq in cpython's Makefile. Looks people try to avoid using ifeq in cpython's Makefile to get it BSD compatible. Am I correct?